src/components/button/button.tsx

```diff
@@ -266,17 +267,17 @@ export class Button implements LabelableComponent, InteractiveComponent {
 
   // act on a requested or nearby form based on type
   private handleClick = (): void => {
-    const { formEl, type } = this;
+    const { type } = this;
 
     if (this.href) {
       return;
     }
 
     // this.type refers to type attribute, not child element type
     if (type === "submit") {
```

Comment:
@jcfranco should we support preventingDefault on click events here or do you think its ok if a user just sets the button type to "button" in those cases?

Reply:
Your suggestion makes sense. I don't think we need to cancel click events.